x-pack/plugins/security_solution/cypress/integration/dashboards/entity_analytics.spec.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login, visit } from '../../tasks/login';
+
+import { ENTITY_ANALYTICS_URL } from '../../urls/navigation';
+
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+import { cleanKibana } from '../../tasks/common';
+import {
+  ENABLE_HOST_RISK_SCORE_BUTTON,
+  ENABLE_USER_RISK_SCORE_BUTTON,
+  HOSTS_DONUT_CHART,
+  HOSTS_TABLE,
+  TABLE_ROWS,
+  USERS_DONUT_CHART,
+  USERS_TABLE,
+} from '../../screens/entity_analytics';
+import { openRiskTableFilterAndSelectTheLowOption } from '../../tasks/host_risk';
+
+describe('Entity Analytics Dashboard', () => {
+  before(() => {
+    cleanKibana();
+    login();
+  });
+
+  describe('Without data', () => {
+    before(() => {
+      visit(ENTITY_ANALYTICS_URL);
+    });
+
+    it('shows enable host risk button', () => {
+      visit(ENTITY_ANALYTICS_URL);
+      cy.get(ENABLE_HOST_RISK_SCORE_BUTTON).should('be.visible');
+    });
+
+    it('shows enable user risk button', () => {
+      cy.get(ENABLE_USER_RISK_SCORE_BUTTON).should('be.visible');
+    });
+  });
+
+  describe('With host risk data', () => {
+    before(() => {
+      esArchiverLoad('risky_hosts');
+      visit(ENTITY_ANALYTICS_URL);
+    });
+
+    after(() => {
+      esArchiverUnload('risky_hosts');
+    });
+
+    it('renders donut chart', () => {
+      cy.get(HOSTS_DONUT_CHART).should('be.visible');
```

Comment:
Not sure if this assertion is really needed taking into account the other one :)